client/test/account_store.spec.js

```diff
@@ -0,0 +1,180 @@
+  });
+
+
+  /*
+   * Problem noticed:
+   *
+   * * Remove mailbox event doesn't do anything.
+   * * getDefault can be used as getByMailbox.
+   */
+  describe('Actions', () => {
+    it('ADD_ACCOUNT_SUCCESS', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.ADD_ACCOUNT_SUCCESS,
+        value: { account: { id: '125' } },
+      });
+      const accounts = accountStore.getAll();
+      assert.equal(accounts.get('125').get('id'), '125');
+      assert.deepEqual(accounts.get('125').get('mailboxes').toObject(), {});
+    });
+    it('EDIT_ACCOUNT_SUCCESS', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.EDIT_ACCOUNT_SUCCESS,
+        value: { rawAccount: { id: '125', login: 'cozy' } },
+      });
+      const accounts = accountStore.getAll();
+      assert.equal(accounts.get('125').get('login'), 'cozy');
+    });
+    it('MAILBOX_CREATE_SUCCESS', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.MAILBOX_CREATE_SUCCESS,
+        value: { id: '125', login: 'cozy', initialized: true },
+      });
+      const accounts = accountStore.getAll();
+      assert.equal(accounts.get('125').get('initialized'), true);
+    });
+    it('MAILBOX_UPDATE_SUCCESS', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.MAILBOX_UPDATE_SUCCESS,
+        value: {
+          id: '125', login: 'cozy', password: 'pass', initialized: true,
+        },
+      });
+      const accounts = accountStore.getAll();
+      assert.equal(accounts.get('125').get('password'), 'pass');
+    });
+    it.skip('REMOVE_ACCOUNT_SUCCESS', () => {
+    });
+    it('MAILBOX_DELETE_SUCCESS', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.MAILBOX_DELETE_SUCCESS,
+        value: { id: '125' },
+      });
+      const accounts = accountStore.getAll();
+      assert.isUndefined(accounts.get('125').get('password'));
+    });
+    it('RECEIVE_MAILBOX_UPDATE', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.RECEIVE_MAILBOX_UPDATE,
+        value: { id: 'b1', label: 'mailbox-edited' },
+      });
+      const accounts = accountStore.getAll();
+      const mailbox = accounts.get('124').get('mailboxes').get('b1');
+      assert.equal(mailbox.get('label'), 'mailbox-edited');
+    });
+  });
+
+  describe('Methods', () => {
+    it.skip('_getMailboxIndex', () => {});
+    it.skip('_setMailboxToImmutable', () => {});
+    it.skip('_initialize', () => {});
+    it.skip('_getByMailbox', () => {});
+    it.skip('_updateMailbox', () => {});
+    it.skip('_updateAccount', () => {});
+    it('getByID', () => {
+      const account = accountStore.getByID('124');
+      assert.equal(account.get('id'), '124');
+    });
+    it('getByMailbox', () => {
+      const account = accountStore.getByMailbox('b1');
+      assert.equal(account.get('id'), '124');
+    });
+    it('getDefault', () => {
+      const account = accountStore.getDefault();
+      assert.equal(account.get('id'), '123');
+    });
+    it('getByLabel', () => {
```

Comment:
@frankrousseau this method isn't used, you can remove this test and remove method from Store (@m4dz: can you confirm?)